store/eth_client.go

```diff
@@ -54,6 +54,30 @@ func (eth *EthClient) GetEthBalance(address common.Address) (float64, error) {
 	return utils.WeiToEth(numWei), nil
 }
 
+func (eth *EthClient) GetERC20Balance(address common.Address, contractAddress common.Address) (*big.Int, error) {
+	result := ""
+	numLinkBigInt := new(big.Int)
+	functionSelector := models.HexToFunctionSelector("0x70a08231") // balanceOf(address)
+	contractAddressBytes, err := utils.HexToBytes(contractAddress.Hex())
+	if err != nil {
+		return nil, err
```

Comment:
Let's replace the 4 line `contractAddressBytes` retrieval clause with `contractAddress.Bytes()`:
https://github.com/ethereum/go-ethereum/blob/master/common/types.go#L161